src/bloaty.cc

```diff
@@ -2166,6 +2196,8 @@ bool DoParseOptions(bool skip_unknown, int* argc, char** argv[],
       }
     } else if (args.TryParseOption("--source-filter", &option)) {
       options->set_source_filter(std::string(option));
+    } else if (args.TryParseOption("--source-map", &option)) {
```

Comment:
Please add documentation for this flag to the usage string and into the user guide.

Reply:
Done - added usage notes to using.md and added the option to the usage string